src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -1250,7 +1250,7 @@ private static bool HadLambdaConversionError(BindingDiagnosticBag diagnostics, A
                         argument.Display,
                         name,
                         method,
-                        parameter);
+                        new FormattedSymbol(parameter, SymbolDisplayFormat.CSharpErrorMessageTypeFormat));
```

Comment:
>new FormattedSymbol(parameter, SymbolDisplayFormat.CSharpErrorMessageTypeFormat) [](http://example.com/codeflow?start=24&length=80)

We need type, let's use type instead of special display format.